app/src/main/res/values/strings.xml

```diff
@@ -7,6 +7,8 @@
   <string name="menu_home">Home</string>
   <string name="menu_help">Help</string>
   <string name="admin_settings">Admin Settings</string>
+  <string name="drawer_open">Open</string>
```

Comment:
For this & the next string: can you indicate these are content descriptions? E.g. ``drawer_open_content_description`` would be a clearer name.

Also, when using TalkBack does 'Open' provide enough context? If you haven't checked yet, you should to make sure the readout label is clear when opening/closing the nav drawer with TalkBack.

Reply:
Updated label, checked talkback and updated  string